src/printer/sygus_print_callback.h

```diff
@@ -154,6 +154,8 @@ class CVC4_PUBLIC SygusEmptyPrintCallback : public SygusPrintCallback
   virtual void toStreamSygus(const Printer* p,
                              std::ostream& out,
                              Expr e) const override;
+  /* empty callback object */
+  static const std::shared_ptr<SygusEmptyPrintCallback> empty_pc;
```

Comment:
Data members (including static ones) should have d_ prefix.

Also, this should be a private member, where it can be accessed by a static public function, say mkEmptyCallback, which returns this pointer.